gamecenter/api/helpers/get.py

```diff
@@ -46,6 +47,23 @@ def get_args():
     return get_args
 
 
+def compare_dates(start, end):
+    try:
+        if start is None:
+            start = DEFAULTS['start_date'].isoformat()
+        else:
+            start = iso8601.parse_date(start)
+        if end is None:
+            end = DEFAULTS['end_date'].isoformat()
+        else:
+            end = iso8601.parse_date(end)
+        if end < start:
+            raise InvalidUsage("The end_date argument must be a date after the start_date argument.")
+    except iso8601.iso8601.ParseError:
+        # This will be caught later
```

Comment:
this isn't to validate that the dates are in proper format, it's just to validate that the start_date and end_date args are in the correct order. Parsing validation will come later